### ISSUE TITLE
Translate

### ISSUE THREAD

**Issue by imtsy** [sumn2u/learn-javascript#3]
I wish it could be translated by many different language.

**Comment by sumn2u** [comment]
> I wish it could be translated by many different language.

True, we wish that too.  We have a volunteer for Turkish translation #1  and are looking for others too.

**Comment by sumn2u** [comment]
[Update] The language support feature has been added  #2 .

**Comment by scheuerkev** [comment]
Hi there ! 
I'm ready to work on French translation if you want 💥

**Comment by sumn2u** [comment]
@scheuerkev That would be awesome. I've assigned you for French translations.

**Comment by hamfree** [comment]
Hi, sumn2u!

And now, i'm ready for working on Spanish translation if you want!

Regards.

**Comment by sumn2u** [comment]
@hamfree Thanks for your interest on translating it to Spanish. I've assigned you for Spanish translations.

**Comment by moustaphaotf** [comment]
Hello @scheuerkev 
Hello @sumn2u 

I'd also like to work on the french translation. Is there a way for us to work together ?
I discovered this project proudly because of the hacktoberfest. If my collaboration would help it will be a pleasure ^

**Comment by sumn2u** [comment]
@moustaphaotf thanks for your interest on translating it in French. I think you can co-ordinate with @scheuerkev to split the chapters and work in parallel.

**Comment by scheuerkev** [comment]
Hello @moustaphaotf !
No problem, with pleasure, I'll submit my PR with the work I've made till here 
When accepted you'll see 5 untranslated folders remaining (which are exercises/ loops/ miscellanous/ objects/ promise-async-await/). Maybe you can work on these for starting. Imo, once we've finished that, we can concentrate on the last chapters written. Some was released quite recently.
Regards,

**Comment by moustaphaotf** [comment]
Thanks @scheuerkev , I also would like to take this work under the "hacktoberfest" label if possible @sumn2u .
I'm taking the work seriously, so if I would be rewarded by a grade with the modest effort I made, I would be gratefull.

**Comment by sumn2u** [comment]
The task has been split into two parts with the "hacktoberfest" tag for easier completion after discussions.

cc/ @scheuerkev @moustaphaotf

**Comment by SarthakJaindebugger** [comment]
@sumn2u do u need translation in any other language? I can do the needful.

**Comment by sumn2u** [comment]
> @sumn2u do u need translation in any other language? I can do the needful.

@SarthakJaindebugger  If you can translate, proofread, or assist current translators in other languages, it would be greatly appreciated.

**Comment by SarthakJaindebugger** [comment]
> > @sumn2u do u need translation in any other language? I can do the needful.
> 
> @SarthakJaindebugger If you can translate, proofread, or assist current translators in other languages, it would be greatly appreciated.

@sumn2u surely, I'll be able to do that. Plz specify in which language u want. Also, plz assign this issue to me.

**Comment by sumn2u** [comment]
@SarthakJaindebugger The content is currently being translated into French by @moustaphaotf and into Spanish by @hamfree. Would you be able to assist there?

**Comment by somuyakhandelwal** [comment]
Dear @sumn2u,

I trust this message finds you in good health. I have come across a document, and I would like to offer my assistance in translating it into Hindi. I discovered this opportunity during **Hacktoberfest**, and I am keen to contribute under that specific tag.

Please consider assigning this task to me, and I will ensure a professional and accurate translation.

Warm regards,
**Somuya Khandelwal**

**Comment by AnshuWorld** [comment]
i want to contribute here please assign me

**Comment by moustaphaotf** [comment]
Thi.j there a still some chapters to translate

**Comment by AnshuWorld** [comment]
I can translate

**Comment by moustaphaotf** [comment]
What languages ?

**Comment by virginus01** [comment]
I am translating to Indonesia, hope am welcomed?

**Comment by sumn2u** [comment]
> I am translating to Indonesia, hope am welcomed?

Of course. Love to have it translated in Indonesian.

**Comment by virginus01** [comment]
Alright. starting it asap.

**Comment by hamfree** [comment]
> @hamfree Thanks for your interest on translating it to Spanish. I've assigned you for Spanish translations.

Hi, again!

I'm finished the translation to spanish. I guess you can view the results in:
https://learn-javascript-es.vercel.app/

How can i do a pull-request?

Thank you!

**Comment by moustaphaotf** [comment]
> How can i do a pull-request?

Hey thank for your awesome work !
It's really great to have you here !
For your contribution to be merged, create a new pull request in https://github.com/sumn2u/learn-javascript/pulls, carefully choose the branch and wait for @sumn2u to merge it!

Right after creating the PR, a preview will be available 🤗

**Comment by hamfree** [comment]
i've just done the PR.

**Comment by nickAP02** [comment]
I would like to continue the french translation

**Comment by moustaphaotf** [comment]
> I would like to continue the french translation

Heyy that's really awesome.
Your help will be very valuable to the project !

**Comment by DimplesY** [comment]
Can I participate in Chinese translation work?

**Comment by sumn2u** [comment]
> Can I participate in Chinese translation work?

Sure.

**Comment by sumn2u** [comment]
@DimplesY I created an issue #231 for the Chinese translation. You can assign it to yourself and start working on it. Let me know if you need any assistance.

**Comment by gramos-encora** [comment]
Hey @sumn2u i just worked on [Issue #28 ](https://github.com/sumn2u/learn-javascript/issues/28), could you create an issue to translate that chapter to spanish and assign it to me, i can work on that too
